shared/src/commonTest/kotlin/co/nimblehq/blisskmmic/data/datastore/core/DataStoreTest.kt

```diff
@@ -38,7 +38,7 @@ class DataStoreTest : TestsWithMocks() {
     }
 
     @Test
-    fun `When saving a token, it sets settings with the correct key and value`() = runTest {
+    fun `When saving a token- it sets settings with the correct key and value`() = runTest {
```

Comment:
Just curious, why you made this change?

Reply:
I ran `./gradlew iosSimulatorArm64Test` to test platform specific code and got this error. 
<img width="701" alt="Screen Shot 2022-12-22 at 10 14 59" src="https://user-images.githubusercontent.com/6356137/209047844-107c58f6-fa9e-4680-9552-a3c588727e56.png">

Reply:
@blyscuit this looks like a really weird issue, even for Android developers as we use comma in the testing function naming all the times, not sure why with your project, we disallow it like this. And this issue doesn't happen before with the existing tests on previous CI runs, right?

Maybe you can check the gradle configuration on the android linting/detekt libraries to make sure the are on the latest versions and it can support this properly 🤔

Reply:
@minhnimble Lucas explains this better than me [here](https://github.com/luongvo/kmm-survey/pull/51#discussion_r1056199814). This was not an issue before because the test was run with which was able to bypass this error somehow, but now that we are running the test against iOS Simulator, with Kotlin, it starts failing.

Reply:
Oh I see, got it now, then maybe we can add a front space for the dash like what @luongvo did for consistency. And please apply for the rest of the test cases 🙏
```suggestion
    fun `When saving a token - it sets settings with the correct key and value`() = runTest {
```

Reply:
Oh I see, got it now, then maybe we can add a front space for the dash like what @luongvo did for consistency. And please apply for the rest of the test cases 🙏
```suggestion
    fun `When saving a token - it sets settings with the correct key and value`() = runTest {
```